fireapp/lib/presentation/shift_request/shift_request_view_model.dart

```diff
@@ -28,6 +31,26 @@ class ShiftRequestViewModel extends FireAppViewModel {
 
   ShiftRequestViewModel(this._shiftRequestRepository);
 
+  void loadMockShiftRequests() async {
```

Comment:
You shouldn't need to differentiate between mock and real data in the viewmodel. Please combine this and the real function